src/components/Dialogs/DialogsContainer.jsx

```diff
@@ -1,30 +1,32 @@
 import React from "react";
 
-import ACTION_TYPES from "../../redux/actionTypes";
-
 import Dialogs from "./Dialogs";
+import { connect } from "react-redux";
+import {
+  updateNewMessageBodyCreator,
+  updateSendMessageCreator,
+} from "./../../redux/actionCreators";
 
-const DialogsContainer = (props) => {
-  const { dialogsPage, dispatch } = props;
-
-  const onSendMessageClick = () => {
-    dispatch({ type: ACTION_TYPES.SEND_MESSAGE });
+let mapStateToProps = (state) => {
```

Comment:
Consider using "const" instead of "let" here.

Reply:
done